graylog2-web-interface/src/components/users/UsersOverview/UserOverviewItem/LoggedInCell.jsx

```diff
@@ -20,19 +19,22 @@ const Td: StyledComponent<{}, ThemeInterface, HTMLTableCellElement> = styled.td`
   text-align: right;
 `;
 
-const LoggedInCell = ({ lastActivity, clientAddress, sessionActive }: Props) => (
+const LoggedInInfo = ({ lastActivity }: { lastActivity: $PropertyType<UserOverview, 'lastActivity'> }) => (
+  <OverlayTrigger trigger={['hover', 'focus']}
+                  placement="right"
+                  overlay={(
+                    <Popover id="session-badge-details" title="Logged in">
+                      <div>Last activity: {lastActivity ? <Timestamp dateTime={lastActivity} relative /> : '-'}</div>
+                    </Popover>
+                  )}
+                  rootClose>
+    <LoggedInIcon active />
+  </OverlayTrigger>
+);
+
+const LoggedInCell = ({ lastActivity, sessionActive }: Props) => (
   <Td>
-    <OverlayTrigger trigger={['hover', 'focus']}
-                    placement="right"
-                    overlay={(
-                      <Popover id="session-badge-details" title={sessionActive ? 'Logged in' : 'Logged out'}>
-                        <div>Last activity: {lastActivity ? <Timestamp dateTime={lastActivity} relative /> : '-'}</div>
-                        <div>Client address: {clientAddress ?? '-'}</div>
-                      </Popover>
-                    )}
-                    rootClose>
-      <LoggedInIcon active={sessionActive} />
-    </OverlayTrigger>
+    {sessionActive ? <LoggedInInfo lastActivity={lastActivity} /> : <LoggedInIcon active={false} />}
```

Comment:
And we still want to show the pop over but only then with the info: "Not logged in"